test/extensions/ordered_list_with_checkboxes.unit

```diff
@@ -45,21 +45,19 @@
 <li>five</li>
 </ol>
 >>> mixed leading spaces
-1.[ ] zero
-2. [ ] one
-3.  [ ] two
-4.   [ ] three
-5.    [ ] four
-6.     [ ] five
+1. [ ] zero
+2.  [ ] one
+3.   [ ] two
+4.    [ ] three
+5.     [ ] four
 <<<
-<p>1.[ ] zero</p>
-<ol start="2" class="contains-task-list">
+<ol class="contains-task-list">
```

Comment:
Only lists starting with 1 to interrupt paragraphs. otherwise it will be treated as a paragraph continuation.
See: https://spec.commonmark.org/0.30/#example-304